External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -1344,31 +1344,94 @@ static private void ShowCalltip(ScintillaControl Sci, int paramIndex, bool force
             else UITools.CallTip.CallTipSetHlt(start + 1, end, true);
         }
 
-        static string[] featStart = new string[] { "/*", "{", "<", "[", "(" };
-        static string[] featEnd = new string[] { "*/", "}", ">", "]", ")" };
-
         static private int FindNearSymbolInFunctDef(string defBody, string symbol, int startAt)
```

Comment:
No I mean the only `symbol` that are looked up in the function definition are `,` and `)`.